packages/admin/src/Pages/Page.php

```diff
@@ -40,15 +40,20 @@ public static function registerNavigationItems(): void
             return;
         }
 
-        Filament::registerNavigationItems([
+        Filament::registerNavigationItems(static::getNavigationItems());
+    }
+
+    public static function getNavigationItems(): array
```

Comment:
Pretty sure they aren't - but these method name changes to resources and pages arent breaking are they? If people have overridden the `registerNavigationItems()` methods.

Reply:
I wouldn't have said so. If you overwrite the method completely, then you won't be using the underlying method. If you are using it, then the behaviour is still the same just extracted.

Reply:
Awesome